code/_onclick/item_attack.dm

```diff
@@ -20,7 +20,7 @@
 	return SendSignal(COMSIG_PARENT_ATTACKBY, W, user, params)
 
 /obj/attackby(obj/item/I, mob/living/user, params)
-	return ..() || I.attack_obj(src, user)
+	return ..() || (attackable && I.attack_obj(src, user))

```

Comment:
Please reword this var because I originally thought it would prevent all of attackby from running